tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/TestReportTest.java

```diff
@@ -62,7 +62,7 @@ public void testWarningForReportTools() throws BallerinaTestException, IOExcepti
         clientLeecher.waitForText(60000);
     }
 
-    @Test ()
+    @Test (enabled = false) // TODO: Enable the test case once the mismatch on line coverage is fixed. issue:#27524
```

Comment:
is this enabled?

Reply:
yes, it has been enabled with the pr -> https://github.com/ballerina-platform/ballerina-lang/commit/8f5e06ceb3cd62f446b2bc4fdd5f0a29a73c277c